e2e/paths/enterOrdersAndDirections_test.js

```diff
@@ -0,0 +1,23 @@
+const config = require('../config.js');
+
+Feature('Enter order and details');
+
+Before((I, caseViewPage) => {
+  I.logInAndCreateCase(config.localAuthorityEmail, config.localAuthorityPassword, config.eventSummary, config.eventSummary);
+  caseViewPage.goToNewActions(config.applicationActions.selectOrders);
+});
+
+Scenario('Select care order', 
+(I, ordersNeededPage) => {
```

Comment:
we dont need to break line so soon, this is what we've been doing:
```suggestion
Scenario('Select care order', (I, ordersNeededPage) => {
```

Reply:
Line 16 is fine however to break line on after scenario string?

Reply:
Line 16 is fine however to break line on after scenario string?